packages/odata-common/src/uri-conversion/odata-uri.ts

```diff
@@ -18,7 +19,11 @@ import { createGetResourcePathForKeys } from './get-resource-path';
 export interface ODataUri<DeSerializersT extends DeSerializers> {
   getExpand<EntityT extends EntityBase>(
     selects: Selectable<EntityT, DeSerializersT>[],
-    expands: Expandable<EntityT, DeSerializersT>[],
+    expands: Expandable<
+      EntityT,
+      DeSerializersT,
+      EntityApi<EntityBase, DeSerializersT>
```

Comment:
Same here - I think if we don't set anything here, it emphasizes, that it is not used here (and can be "anything")